.cirrus.yml

```diff
@@ -113,43 +132,104 @@ validate_task:
   setup_script: *setup
   main_script: *main
 
+
+validate_aarch64_task:
+  alias: "validate_aarch64"
+  depends_on:
+    - "build_aarch64"
+  ec2_instance: *standard_build_ec2_aarch64
+  cargo_cache: &ro_cargo_cache_aarch64
+    <<: *cargo_cache_aarch64
+    reupload_on_changes: false
+  targets_cache: &ro_targets_cache_aarch64
+    <<: *targets_cache_aarch64
+    reupload_on_changes: false
+  bin_cache: &ro_bin_cache_aarch64
+    <<: *bin_cache_aarch64
+    reupload_on_changes: false
+  setup_script: *setup
+  main_script: *main
+
+
 verify_vendor_task:
   alias: "verify_vendor"
   depends_on:
     - "build"
+  gce_instance: *standard_gce_x86_64
   cargo_cache: *ro_cargo_cache
   targets_cache: *ro_targets_cache
   bin_cache: *ro_bin_cache
   setup_script: *setup
   main_script: *main
 
+
+verify_vendor_aarch64_task:
+  alias: "verify_vendor_aarch64"
+  depends_on:
+    - "build_aarch64"
+  ec2_instance: *standard_build_ec2_aarch64
+  cargo_cache: *ro_cargo_cache_aarch64
+  targets_cache: *ro_targets_cache_aarch64
+  bin_cache: *ro_bin_cache_aarch64
+  setup_script: *setup
+  main_script: *main
+
+
 unit_task:
   alias: "unit"
   depends_on:
     - "build"
+  gce_instance: *standard_gce_x86_64
   cargo_cache: *ro_cargo_cache
   targets_cache: *ro_targets_cache
   bin_cache: *ro_bin_cache
   setup_script: *setup
   main_script: *main
 
+
+unit_aarch64_task:
+  alias: "unit_aarch64"
+  depends_on:
+    - "build_aarch64"
+  ec2_instance: *standard_build_ec2_aarch64
+  cargo_cache: *ro_cargo_cache_aarch64
+  targets_cache: *ro_targets_cache_aarch64
+  bin_cache: *ro_bin_cache_aarch64
+  setup_script: *setup
+  main_script: *main
+
+
 integration_task:
   alias: "integration"
   depends_on:
     - "unit"
+  gce_instance: *standard_gce_x86_64
   cargo_cache: *ro_cargo_cache
   targets_cache: *ro_targets_cache
   bin_cache: *ro_bin_cache
   setup_script: *setup
   main_script: *main
 
 
+integration_aarch64_task:
+  alias: "integration_aarch64"
+  depends_on:
+    - "unit_aarch64"
+  ec2_instance: *standard_build_ec2_aarch64
+  cargo_cache: *ro_cargo_cache_aarch64
+  targets_cache: *ro_targets_cache_aarch64
+  bin_cache: *ro_bin_cache_aarch64
+  setup_script: *setup
+  main_script: *main
+
+
 # This task is critical.  It updates the "last-used by" timestamp stored
 # in metadata for all VM images.  This mechanism functions in tandem with
 # an out-of-band pruning operation to remove disused VM images.
 meta_task:
     alias: meta
     name: "VM img. keepalive"
+    gce_instance: *standard_gce_x86_64
```

Comment:
Oh no, this is wrong.  The meta_task runs inside a specific container image.

Reply:
ah ok

Reply:
done.